<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图像显示区域:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCamera">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>启动程序</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>421</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>识别程序状态输出:</string>
     </property>
     <widget class="QTreeWidget" name="treeWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>网络</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>识别程序状态</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>381</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>识别结果输出区:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>39</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>目标ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>目标数量</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Insulator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ShellBroken</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FlashOver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NoIssues</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>39</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTrans">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>607</y>
      <width>14</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>208</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>607</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>604</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>208</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="CameraWidget" name="cameraWidget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>29</y>
      <width>751</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCamera">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>630</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendfile">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>650</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>sendfile</string>
    </property>
   </widget>
   <widget class="QLabel" name="arrow">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>600</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header>/home/hy/visual_window519/camerawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
